test/e2e/page-objects/pages/home/homepage.ts

```diff
@@ -257,7 +257,7 @@ class HomePage {
     if (localNode) {
       expectedBalance = (await localNode.getBalance(address)).toString();
     } else {
-      expectedBalance = '0';
```

Comment:
we set default value as '25ETH', this way, we can skip passing the local node server in most of the specs, as it's already the default value, to simplify